oneflow/core/ep/include/primitive/where.h

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 The OneFlow Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#ifndef ONEFLOW_CORE_EP_PRIMITIVE_WHERE_H_
+#define ONEFLOW_CORE_EP_PRIMITIVE_WHERE_H_
+
+#include "oneflow/core/ep/include/primitive/primitive.h"
+
+namespace oneflow {
+namespace ep {
+namespace primitive {
+
+class Where : public Primitive {
+ public:
+  OF_DISALLOW_COPY_AND_MOVE(Where);
+  Where() = default;
+  ~Where() override = default;
+
+  virtual void Launch(Stream* stream, DataType cond_type, size_t cond_ndim,
```

Comment:
```suggestion
  virtual void Launch(Stream* stream, DataType cond_type, size_t num_cond_dims,
```
这个模块下明明风格尽量统一，参考其他 https://github.com/Oneflow-Inc/oneflow/blob/55bdce625e93f73fbf774157a9e9ea43d473e1c0/oneflow/core/ep/include/primitive/broadcast_elementwise_binary.h#L34